examples/apexlog/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"errors"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/airbrake/gobrake/v5/apexlog"
+	"github.com/apex/log"
+)
+
+func main() {
+	// Initialize the airbrake notifier
+	airbrake := gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+		ProjectId:   105138,                             // use the ProjectId of you project
+		ProjectKey:  "fd04e13d806a90f96614ad8e529b2822", // use the ProjectKey of you project
+		Environment: "production",
+	})
+
+	// Close the airbrake notifier
+	defer airbrake.Close()
+
+	// Create the airbrake handler for "github.com/apex/log"
+	// Insert the airbrake notifier instance and set the severity level.
+	// The acceptable severity levels are "debug" "info", "warn", "error", and "fatal".
+	// If the severity level is not recognized, "error" severity is used.
+	airbrakeHandler := apexlog.New(airbrake, "error")
+
+	log.SetLevel(log.DebugLevel)
+
+	// Set the airbrake handler in the log
+	log.SetHandler(airbrakeHandler)
+
+	ctx := log.WithFields(log.Fields{
+		"file": "something.png",
+		"type": "image/png",
+		"user": "tobi",
+	})
+
+	for range time.Tick(time.Millisecond * 500) {
```

Comment:
Why do we need a timer?

Reply:
This example is use used for the apex/log package(the one they used in their package), so we can show that the middleware will work without making any modifications to apex/log and it will be easier for developers to understand who are using apex/log already.

Reply:
I gave this a run locally and I sent 160 errors to Airbrake within ~15 seconds. I don't think it's a good idea to run this on a loop because our users can leave this running for some time and then they will run out of their error quota or will have to pay more than needed.

I think we need to print these messages without a loop and make the program exit by itself so that we won't waste error quota.